setup.py

```diff
@@ -1,16 +1,18 @@
 import imp
 import os
+import json
 from setuptools import setup, find_packages
 
-version = imp.load_source(
-    'version', os.path.join('caravel', 'version.py'))
+BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(BASE_DIR, 'caravel/static/assets/package.json')) as package_file:
```

Comment:
that makes pythonistas' sad :snake: but makes sense from a maintability point of view. The patch will break windows users though: `os.path.join(BASE_DIR, 'caravel', 'static', 'assets', 'package.json')` will do